File: STRIVER/video/vid-5/pattern15/p15.py
# ABCDE
# ABCD
# ABC
# AB
# A

def p15(row:int)->None:

    for i in range(row,-1,-1):
        for j in range(ord('A'), ord('A') + i + 1):
            print(chr(j),end="")
        print("")

row = 4
p15(row)

File: STRIVER/video/vid-5 (Patterns)/pattern15/p15.cpp
// ABCDE
// ABCD
// ABC
// AB
// A

#include <iostream>
using namespace std;

int main()
{
    int row = 4;

    for (int i = row; i >= 0; i--)
    { 
        for (char j = 'A'; j<= 'A'+ i; j++){
            cout<<j;
        }
        cout<<endl;
    }

    return 0;
}